build/rollup.js

```diff
@@ -0,0 +1,68 @@
+const rollup             = require('rollup');
+const uglify             = require('rollup-plugin-uglify');
+const babel              = require('rollup-plugin-babel');
+const replace            = require('rollup-plugin-replace');
+const eslint             = require('rollup-plugin-eslint');
+const conditional        = require('rollup-plugin-conditional');
+const filesize           = require('rollup-plugin-filesize');
+const resolve            = require('rollup-plugin-node-resolve');
+const commonjs           = require('rollup-plugin-commonjs');
+const visualizer         = require('rollup-plugin-visualizer');
+const builtins           = require('rollup-plugin-node-builtins');
+const globals            = require('rollup-plugin-node-globals');
+const skipCommentsCustom = require('./uglify-skip-comments');
+const generateBanner     = require('./generate-banner');
+const generateBundleName = require('./generate-bundle-name');
+const pkg                = require('../package.json');
+
+
+const env    = process.env.NODE_ENV || 'development';
+const isProd = env === 'production';
+
+
+let promise = Promise.resolve();
+
+['cjs', 'umd'].forEach((format) => {
+  promise = promise.then(() => rollup.rollup({
+    entry  : 'lib/index.js',
+    context: 'global',
+    plugins: [
+      conditional(format === 'cjs', [globals(), builtins()]),
+      eslint(),
+      resolve({
+        jsnext        : format === 'umd',
+        main          : true,
+        browser       : format === 'umd',
+        preferBuiltins: true
+      }),
+      commonjs({
+        namedExports: {
+          'node_modules/qs/lib/index.js': ['stringify']
+          // 'node_modules/whatwg-fetch/fetch.js': ['default']
```

Comment:
Just to be sure, this line was necessary for the `whatwg-fetch` polyfill to work. Have you check if it is necessary with `isomorphic-fetch`?

I assume it does work because `isomorphic-fetch` is already on the build 😅